tests/std/tests/P0645R10_text_formatting_formatting/test.cpp

```diff
@@ -60,6 +60,45 @@ void throw_helper(const charT* fmt, const Args&... vals) {
     }
 }
 
+template <class back_inserter>
+struct move_only_back_inserter {
+    back_inserter it;
+    using difference_type = typename back_inserter::difference_type;
+
+    bool moved_from = false;

```

Comment:
Why do we need to have this member and do the asserts as we go, instead of just making the type move only and immediately aborting if we copy it?

Reply:
oh, it tracks double moves, I see.